test/util/fixtures/IocMapUtility.java

```diff
@@ -0,0 +1,10 @@
+package util.fixtures;
+
+public final class IocMapUtility {
```

Comment:
Rename this to IocFixtures please use name

Reply:
solved